Added FAQ for maximum number of browser tabs and email section removal (#348)
seqeralabs/nf-tower-docs: docs/faqs.md

To reestablish email connectivity, please follow the instructions at [https://support.google.com/accounts/answer/3466521](https://support.google.com/accounts/answer/3466521) to provision an app password. Update your `TOWER_SMTP_PASSWORD` environment variable with the app password, and restart the application.

**<p data-question>Q: Can I disable/remove the email section from the login page?</p>**

No. At the moment, there’s no way to remove the email section from the login page. We recommend using the trustedEmail config on your tower.yml file to prohibit access from all users like below. 
```yaml
tower: 
  trustedEmails:
    - 'noone@nowhere.org'
```

All users not part of the trustedEmail will undergo approval process on Profile -> Admin -> Users page. Other clients have used this as a backup method if the SSO becomes unavailable.

Note: The change above requires your containers to be rebuilt (ie, docker-compose down). Make sure your database is not ephemeral before issuing the docker-compose down command. Please refer to [this page](https://install.tower.nf/latest/docker-compose/) for more information. 

> To ensure that previous users logged in using the email before, an Admin of the organization should remove said users from the Profile -> Admin -> Users list. This will essentially restart the approval process before they are allowed to log in via email.

Reviewer: ```suggestion
An admin of the organization should remove users that have previously logged in via (untrusted) email from the Profile -> Admin -> Users list. This will restart the approval process before they can log in via email.
```
